src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -18,23 +18,26 @@
 import io.cdap.e2e.pages.actions.CdfBigQueryPropertiesActions;
 import io.cdap.e2e.pages.actions.CdfStudioActions;
 import io.cdap.e2e.pages.locators.CdfStudioLocators;
-import io.cdap.e2e.utils.BigQueryClient;
-import io.cdap.e2e.utils.ConstantsUtil;
-import io.cdap.e2e.utils.ElementHelper;
-import io.cdap.e2e.utils.PluginPropertyUtils;
-import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.e2e.utils.*;
```

Comment:
Make sure existing imports are not removed, and since * is added you will get the check style issue. Please add in the order it was and check once if any check style issues occurs resolve it.